Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/formatter/FormatterInformationLegalHelper.kt

```diff
@@ -15,9 +15,10 @@ import de.rki.coronawarnapp.util.device.DefaultSystemInfoProvider
  */
 
 fun formatVisibilityLanguageBased(
+    context: Context,
```

Comment:
Same here, extention of Context

Reply:
Not sure about this..
Methods of Formatter are meant to be used in layouts and extension funs can't be used there